recipes/test_openssh.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+#
+# Cookbook Name:: aws-parallelcluster
+# Recipe:: test_openssh
+#
+# Copyright 2013-2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with the
+# License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Verifies that ssh_target_checker.sh contains the correct VPC CIDR list
+check_ssh_target_checker_vpc_cidr_list('/usr/bin/ssh_target_checker.sh', get_vpc_cidr_list)
```

Comment:
this recipe doesn't appear to be included anywhere

Reply:
Fixed